File: 5_delete_obj_property.py
class Fruit: 
    def __init__(self, name, color): 
        self.name = name 
        self.color = color 
    def show(self): 
        print('Name: ', self.name, 'Color: ', self.color)
#create an object of the class Fruit
obj = Fruit('Apple', 'Red')

#delete the Object property
del obj.name
#Accessing obj property after deleting it will throw an error

print(obj.name)
# output: AttributeError: 'Fruit' object has no attribute 'name'

File: 5_delete_obj_property.php
<?php 
class Fruit {
    private $name;
    private $color;

    public function __construct($name, $color) {
        $this->name = $name;
        $this->color = $color;
    }

    public function show() {
        echo 'Name: ' . $this->name . 'Color: ' . $this->color;
    }
}

// create an object of the class Fruit
$obj = new Fruit('Apple', 'Red');

// delete the Object property
unset($obj->name);

// accessing $obj property after deleting it will throw an error
echo $obj->name;
// output: PHP Fatal error:  Cannot access property name on an object of class Fruit in ...